modules/eventing/pages/eventing-examples-rest-via-curl-get.adoc

```diff
@@ -0,0 +1,404 @@
+
+. [Optional] Wait until the next morning and Click "Log" link for Deployed Function *external_rest_via_curl_get* to view the activity.  The code triggers the initial *cur_recurring_timer::1* in 30 seconds, but for requests 2-N it switche to a daily basis
+** Here we see the timer fired and executed the callback *RecurringTimerCallback* near our scheduled time for the next day and re-arming as expected.
++
+----
+2020-08-07T00:00:01.021-07:00 [INFO] "Finish CreateRecurringTimer (local time) nextSchedule" "2020-08-07T00:00:00.000"
+2020-08-07T00:00:01.485-07:00 [INFO] "From CreateRecurringTimer: creating timer" "via_callback" "recurring_timer::1"
+2020-08-07T00:00:01.487-07:00 [INFO] "USER FUNCTION DONE OKAY (curl 200 took 490 ms.)"
+2020-08-07T00:00:00.003-07:00 [INFO] "From RecurringTimerCallback: timer fired" {"id":"recurring_timer::1","mode":"via_callback"}
+----
++
+[Optional] Wait until the next morning and Click "Log" link for Deployed Function *external_rest_via_curl_get* to view the activity.  
+Now check the results of the callback, access the *Couchbase Web Console* > *Buckets* page and click the *Documents* link of the *source* bucket.
+** Edit the new output status document *cur_recurring_timer::1* (note the last_update field is in UTC we are 7 hours behing UTC) and you will see the data written by the Timer's callback:
++
+----
+{
+  "last_update_loc": "2020-08-07T00:00:01.021",
+  "last_update_utc": "2020-08-07T07:00:01.021Z",
+  "apiReqDateUtc": "2020-08-06",
+  "curl_success": true,
+  "valid": true,
+  "curl_time_ms": 490
+}
+----
+** Click *Cancel* to close the editor.
+
+== *Test 2: Cancel the Recurring Timer:*
+
+. Access the *Couchbase Web Console* > *Buckets* page and click the *Documents* link of the *source* bucket.
+** Edit the control document *recurring_timer::1* it should look like:
++
+----
+{
+  "type": "recurring_timer",
+  "id": 1,
+  "active": true
+}
+----
++
+Now change "a_number" to 2 to create a mutation, then Click *Save*.  The control document is now:
++
+----
+{
+  "type": "recurring_timer",
+  "id": 1,
+  "active": false
+}
+----
+
+. Access the *Couchbase Web Console* > *Eventing* page and click on the Function *external_rest_via_curl_get* then Click the "Log" link for Deployed Function *external_rest_via_curl_get* to view the activity.  
+** Here we see from the Application log that we canceled the sequence, the recurring timer has stopped.
++
+----
+2020-08-07T10:47:05.150-07:00 [INFO] "From OnUpdate: canceled active Timer, doc.active" false "recurring_timer::1"
+----
+
+*Cleanup*:
+
+Cleanup, go to the Eventing portion of the UI and undeploy the Function *external_rest_via_curl_get*, this will remove the 2048 documents for each function from the 'metadata' bucket (in the Bucket view of the UI). Remember you may only delete the 'metadata' bucket if there are no deployed Eventing functions.
```

Comment:
```suggestion
To clean up, go to the Eventing portion of the UI and undeploy the Function *external_rest_via_curl_get*; this will remove the 2048 documents for each function from the 'metadata' bucket (in the Bucket view of the UI). Remember you may only delete the 'metadata' bucket if there are no deployed Eventing functions.
```